dogs.html

```diff
@@ -0,0 +1,132 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Adoptable Dogs</title>
+    <link rel="stylesheet" type="text/css" href="styles/styles.css">
+  </head>
+  <body>
+    <header class="header">
+      <img class="header-img" src="images/adoptadog-logo.svg"  alt="Logo">
+      <h1 class="header-text">Seattle's premier dog adoption agency</h1>
+    </header>
+  <section class="navbar">
+    <nav>
+      <div class="nav">
+      <a class="header-links" href="index.html">Home</a>
+      <a>/</a>
+      <a class="header-links" href="dogs.html">Dogs</a>
+      <a>/</a>
+      <a class="header-links" href="blog.html">Blog</a>
+      </div>
+    </nav>
+  </section>
+
+
+    <h2> Here are all of our dogs!</h2>
+    <div class="container">
+      <div class="box">
```

Comment:
Think of a more semantic name for this class, it's always good to use names that make sense to what we're working with.